### ISSUE TITLE
Resuming Dreambooth training from last saved state does not train further.

### ISSUE THREAD

**Issue by irldoggo** [bmaltais/kohya_ss#3103]
Howdy, I have experienced this phenomenon across multiple Dreambooth runs now. Upon resuming from a last checkpoint state, any further training does not seem to apply to the model.

For example I train 12 Dreambooth Epochs with "save last state" checked. - Run 1
I start a new run from the Run 1 DB save state for 6 more Epochs. - Run 2

Upon testing via XY plot afterwards the outputs of Run 1 and Run 2 are identical, as if no further training has occurred even though there was no error and it does indeed seem like the training runs its full duration with full GPU utilization.

I am on the latest pull from: sd3-flux.1
I am using a fresh venv with python 3.10.13 with all requirements up to date.

Here is roughly the settings I am using for training.
(I am manually calling v_param as I cannot toggle both SDXL and V_param options simultaneously in the GUI.)

`{ "adaptive_noise_scale": 0, "additional_parameters": "--zero_terminal_snr --v_parameterization",  "ae": "", "apply_t5_attn_mask": false, "async_upload": false, "blocks_to_swap": 0, "blockwise_fused_optimizers": false, "bucket_no_upscale": false, "bucket_reso_steps": 64, "cache_latents": true, "cache_latents_to_disk": true, "caption_dropout_every_n_epochs": 0, "caption_dropout_rate": 0, "caption_extension": ".txt", "clip_g": "", "clip_l": "", "clip_skip": 1, "color_aug": false, "cpu_offload_checkpointing": false, "dataset_config": "", "debiased_estimation_loss": false, "disable_mmap_load_safetensors": false, "discrete_flow_shift": 3, "double_blocks_to_swap": 0, "dynamo_backend": "no", "dynamo_mode": "default", "dynamo_use_dynamic": false, "dynamo_use_fullgraph": false, "enable_bucket": true, "epoch": 24, "extra_accelerate_launch_args": "", "flip_aug": false, "flux1_cache_text_encoder_outputs": false, "flux1_cache_text_encoder_outputs_to_disk": false, "flux1_checkbox": false, "flux1_clip_l": "", "flux1_t5xxl": "", "flux_fused_backward_pass": false, "fp8_base": false, "full_bf16": false, "full_fp16": false, "fused_backward_pass": true, "fused_optimizer_groups": 0, "gpu_ids": "", "gradient_accumulation_steps": 1, "gradient_checkpointing": true, "guidance_scale": 3.5, "huber_c": 0.1, "huber_scale": 1, "huber_schedule": "snr", "huggingface_path_in_repo": "", "huggingface_repo_id": "", "huggingface_repo_type": "", "huggingface_repo_visibility": "", "huggingface_token": "", "ip_noise_gamma": 0, "ip_noise_gamma_random_strength": false, "keep_tokens": 1, "learning_rate": 6e-05, "learning_rate_te": 0, "learning_rate_te1": 1.5e-06, "learning_rate_te2": 1.5e-06, "log_config": false, "log_tracker_config": "", "log_tracker_name": "", "log_with": "", "logging_dir": "", "logit_mean": 0, "logit_std": 1, "loss_type": "l2", "lr_scheduler": "cosine_with_restarts", "lr_scheduler_args": "", "lr_scheduler_num_cycles": 3, "lr_scheduler_power": 1, "lr_scheduler_type": "", "lr_warmup": 11, "lr_warmup_steps": 0, "main_process_port": 0, "masked_loss": false, "max_bucket_reso": 2048, "max_data_loader_n_workers": 0, "max_resolution": "1024,1024", "max_timestep": 1000, "max_token_length": 225, "max_train_epochs": 6, "max_train_steps": 0, "mem_eff_attn": false, "mem_eff_save": false, "metadata_author": "", "metadata_description": "", "metadata_license": "", "metadata_tags": "", "metadata_title": "", "min_bucket_reso": 640, "min_snr_gamma": 5, "min_timestep": 0, "mixed_precision": "bf16", "mode_scale": 1.29, "model_list": "custom", "model_prediction_type": "sigma_scaled", "multi_gpu": false, "multires_noise_discount": 0.3, "multires_noise_iterations": 0, "no_token_padding": false, "noise_offset": 0, "noise_offset_random_strength": false, "noise_offset_type": "Original", "num_cpu_threads_per_process": 2, "num_machines": 1, "num_processes": 1, "optimizer": "Adafactor", "optimizer_args": "scale_parameter=False relative_step=False warmup_init=False", "output_dir": "/ALPHA/Train/run6_reg-E24", "output_name": "3dstyle_reg_2", "persistent_data_loader_workers": false, "pretrained_model_name_or_path": "/ALPHA/Train/run6_reg-E24/last.safetensors", "prior_loss_weight": 0.5, "random_crop": false, "reg_data_dir": "/ALPHA/Train/workspace/reg", "resume": "", "resume_from_huggingface": "", "sample_every_n_epochs": 0, "sample_every_n_steps": 0, "sample_prompts": "", "sample_sampler": "euler_a", "save_clip": false, "save_every_n_epochs": 2, "save_every_n_steps": 0, "save_last_n_epochs": 0, "save_last_n_epochs_state": 0, "save_last_n_steps": 6, "save_last_n_steps_state": 0, "save_model_as": "safetensors", "save_precision": "bf16", "save_state": false, "save_state_on_train_end": true, "save_state_to_huggingface": false, "save_t5xxl": false, "scale_v_pred_loss_like_noise_pred": false, "sd3_cache_text_encoder_outputs": false, "sd3_cache_text_encoder_outputs_to_disk": false, "sd3_checkbox": false, "sd3_text_encoder_batch_size": 1, "sdxl": true, "sdxl_cache_text_encoder_outputs": false, "sdxl_no_half_vae": true, "seed": 1337, "shuffle_caption": true, "single_blocks_to_swap": 0, "skip_cache_check": false, "split_mode": false, "stop_text_encoder_training": 20, "t5xxl": "", "t5xxl_device": "", "t5xxl_dtype": "bf16", "t5xxl_max_token_length": 512, "timestep_sampling": "sigma", "train_batch_size": 4, "train_blocks": "all", "train_data_dir": "/ALPHA/Train/workspace/dataset", "v2": false, "v_parameterization": false, "v_pred_like_loss": 0, "vae": "", "vae_batch_size": 0, "wandb_api_key": "", "wandb_run_name": "", "weighted_captions": false, "weighting_scheme": "logit_normal", "xformers": "xformers" }`

Any insight on this would be greatly appreciated. Thanks.
